Scripts/workload-install.ps1

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env pwsh
```

Comment:
This script sort of works on macOS. I did not test Linux. It won't work on Windows as that requires either MSI files (I may look at this in the future) or manually copying of things.